gradle/libs.versions.toml

```diff
@@ -0,0 +1,66 @@
+[versions]
+groovy = "3.0.10"
+jcache = "1.1.1"
+logback-classic = "1.2.11"
+metrics-core = "4.2.9"
+micrometer = "1.8.3"
+new-relic-registry = "0.8.0"
+reflections = "0.10.2"
+validation = "2.0.1.Final"
+
+# Micronaut
+micronaut = "3.4.3"
+micronaut-docs = "2.0.0"
+
+
+[libraries]
+# Micronaut
+micronaut-aop = { module = "io.micronaut:micronaut-aop", version.ref = "micronaut" }
+micronaut-cache-core = { module = "io.micronaut.cache:micronaut-cache-core" }
+micronaut-docs-asciidoc-config-props = { module = "io.micronaut.docs:micronaut-docs-asciidoc-config-props", version.ref = "micronaut-docs" }
+micronaut-function-web = { module = "io.micronaut:micronaut-function-web" }
+micronaut-graal = { module = "io.micronaut:micronaut-graal" }
+micronaut-http-client = { module = "io.micronaut:micronaut-http-client" }
+micronaut-http-server-netty = { module = "io.micronaut:micronaut-http-server-netty" }
+micronaut-inject = { module = "io.micronaut:micronaut-inject", version.ref = "micronaut" }
+micronaut-inject-groovy = { module = "io.micronaut:micronaut-inject-groovy" }
+micronaut-inject-groovy-test = { module = "io.micronaut:micronaut-inject-groovy-test", version.ref = "micronaut" }
+micronaut-inject-java = { module = "io.micronaut:micronaut-inject-java" }
+micronaut-inject-java-test = { module = "io.micronaut:micronaut-inject-java-test", version.ref = "micronaut" }
+micronaut-jdbc = { module = "io.micronaut.sql:micronaut-jdbc" }
+micronaut-management = { module = "io.micronaut:micronaut-management" }
+micronaut-rxjava2 = { module = "io.micronaut.rxjava2:micronaut-rxjava2" }
+
+cache-api = { module = "javax.cache:cache-api", version.ref = "jcache" }
+groovy-json = { module = "org.codehaus.groovy:groovy-json", version.ref = "groovy" }
+grpc-api = { module = "io.grpc:grpc-api" }
+logback-classic = { module = "ch.qos.logback:logback-classic", version.ref = "logback-classic" }
+metrics-core = { module = "io.dropwizard.metrics:metrics-core", version.ref = "metrics-core" }
+micrometer-core = { module = "io.micrometer:micrometer-core", version.ref = "micrometer" }
```

Comment:
Should those `micrometer-xxx` dependencies be actually managed in the Micronaut Micrometer BOM?

Reply:
Maybe even better to use BOM directly: https://search.maven.org/artifact/io.micrometer/micrometer-bom/1.8.5/pom